tests/system/action/meeting_mediafile/test_delete.py

```diff
@@ -0,0 +1,26 @@
+from tests.system.action.base import BaseActionTestCase
+
+
+class MeetingMediafileDelete(BaseActionTestCase):
+    def test_delete(self) -> None:
+        self.create_meeting()
+        self.set_models(
```

Comment:
Please write a test with more or less all fields, especially the relations list_of_speakers, assignments , ... and show that they are handled as expected

Reply:
Done